resources/systems/linux/es_find_rules.xml

```diff
@@ -810,6 +810,7 @@
             <entry>ruffle</entry>
         </rule>
         <rule type="staticpath">
+            <entry>/app/bin/ruffle</entry>
```

Comment:
I would put:

```xml
<entry>/app/bin/ruffle</entry> <!-- RetroDECK -->
```

To show it's our custom entry, it helps when we have to compare the incoming edits.